.github/CODEOWNERS

```diff
@@ -0,0 +1,29 @@
+# See https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# Global owners, default owners for everything in the repo
+*                                               @swissspidy
+
+# All server-side logic.
+*.php                                           @swissspidy @spacedmonkey @miina
+composer.json                                   @swissspidy @spacedmonkey @miina
+phpcs.xml                                       @swissspidy @spacedmonkey @miina
+phpstan.neon.dist                               @swissspidy @spacedmonkey @miina
+phpunit.xml.dist                                @swissspidy @spacedmonkey @miina
+
+# Infrastructure
+.github/                                        @swissspidy
+.storybook/                                     @swissspidy
+bin/                                            @swissspidy
```

Comment:
Perhaps test infrastructure too, as they don't contain the actual tests, just the setup?

```
tests/                                            @swissspidy
```

Reply:
Not necessarily true for e2e tests and PHP unit tests